libs/sdk-core/src/moon_pay/moon_pay_config.rs

```diff
@@ -0,0 +1,25 @@
+#[derive(Clone)]
+pub struct MoonPayConfig {
```

Comment:
I would rather not expose the configuration (not even out of the moonpay module) and not to the user.
All the values are hard coded eventually and I am not sure we want the user to change them.
The only exception that we need to think about is how to deal with the API key.

Reply:
I'm thinking of using the same way as breez mobile, what do you think?